src/shared/user-menu.jsx

```diff
@@ -52,6 +54,15 @@ export default class UserMenu extends React.Component {
     this.setState({ loginModalVisible: true })
   }
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Please don't disable eslint rules unless you have an extremely good reason for doing so. If you find that eslint is warning it's generally a sign that there is a problem with your code or it is more verbose than necessary-- in this case there is no need to have an unused dispatch parameter and you can just leave it out.

Reply:
ok